sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_shared_access_signature.py

```diff
@@ -194,6 +192,8 @@ def generate_directory_sas(
         credential,  # type: Union[str, UserDelegationKey]
         permission=None,  # type: Optional[Union[DirectorySasPermissions, str]]
         expiry=None,  # type: Optional[Union[datetime, str]]
+        start=None,  # type: Optional[Union[datetime, str]]
+        policy_id=None,  # type: Optional[str]
```

Comment:
These should be keywords and pass straight into `generate_blob_sas` through `kwargs` like we did for file_system -> container.